### PR TITLE
Check if people are unreachable

### PR DESCRIPTION
**Pull request by SnazzyPine25**

### CODE REVIEW

**Comment by z64** [comment]
## RIP in many pepperonos

Just wanted to write my thoughts out here on this before I go through with testing & accepting this PR.

I had asked you in the dev discord if you were hosting KekBot, because I was curious how you got the idea to do this, why you thought it would benefit KekBot.

My first guess is going to be that this will keep the `db` clean from "dead users", that at a certain point, would just waste performance iterating over doing the various things KekBot does. Again, this is just my guess, I don't know why you thought of this for sure.

So on that token - I will point out that I think this is great for if and when KekBot gets bigger in userbase. However, I'll briefly talk about this scenario that happened recently that makes me reluctant to implement this; 

Recently, someone in one of KekBot's servers, a member had a falling out with another member that resulted in him briefly leaving the server. In this case, he would have been wiped from the db upon leaving, and when he came back, all his data in KekBot would be gone and he'd have to start from the ground up.

I would like to be more forgiving in this scenario, though of course it's probably a rare occurrence. 


## Resurrection

If it *is* a performance thing as I described, I might be more inclined to make a separate collection in the db called `graveyard` that these users are moved into and are removed from KekBot's interactions. If they were to rejoin, they could be restored to the working `users` section with something like `:ressurect [id]`.

To be clear - upon "death", the users `collectibles` will be released. The things I'm interested in putting in `graveyard` are their `currency` savings and (eventually) their userlog, which will cover achievements and whatnot. This way all is not lost and they can come back and continue progressing with the bot.

## TL;DR

I'll accept this for now, because I like it, but I would rather archive users in a `graveyard` than delete them.
